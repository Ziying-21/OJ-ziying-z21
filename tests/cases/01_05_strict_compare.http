###
# Send request:
POST http://127.0.0.1:12345/jobs HTTP/1.1
content-type: application/json

{"contest_id":0,"language":"Rust","problem_id":0,"source_code":"fn main() { print!(\"Hello World!\"); }","user_id":0}
###
# Got response:
# HTTP 200
# content-length: 375
# access-control-allow-credentials: true
# vary: Origin, Access-Control-Request-Method, Access-Control-Request-Headers
# content-type: application/json
# access-control-expose-headers: content-type
# date: Fri, 09 Sep 2022 02:18:53 GMT
# 
# {"cases":[{"id":0,"info":"","memory":0,"result":"Waiting","time":0},{"id":1,"info":"","memory":0,"result":"Waiting","time":0}],"created_time":"2022-09-09T02:18:53.829Z","id":0,"result":"Waiting","score":0.0,"state":"Queueing","submission":{"contest_id":0,"language":"Rust","problem_id":0,"source_code":"fn main() { print!(\"Hello World!\"); }","user_id":0},"updated_time":""}
###
# Send request:
GET http://127.0.0.1:12345/jobs/0 HTTP/1.1


###
# Got response:
# HTTP 200
# content-length: 431
# access-control-allow-credentials: true
# vary: Origin, Access-Control-Request-Method, Access-Control-Request-Headers
# content-type: application/json
# access-control-expose-headers: content-type
# date: Fri, 09 Sep 2022 02:18:54 GMT
# 
# {"cases":[{"id":0,"info":"","memory":0,"result":"Compilation Success","time":124672},{"id":1,"info":"","memory":0,"result":"Wrong Answer","time":313895}],"created_time":"2022-09-09T02:18:53.829Z","id":0,"result":"Wrong Answer","score":0.0,"state":"Finished","submission":{"contest_id":0,"language":"Rust","problem_id":0,"source_code":"fn main() { print!(\"Hello World!\"); }","user_id":0},"updated_time":"2022-09-09T02:18:54.300Z"}
