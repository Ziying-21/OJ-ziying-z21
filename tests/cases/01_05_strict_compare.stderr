[2022-09-09T02:18:52Z INFO  actix_server::builder] Starting 8 workers
[2022-09-09T02:18:52Z INFO  actix_server::server] Actix runtime found; starting in Actix runtime
[2022-09-09T02:18:53Z INFO  actix_web::middleware::logger] 127.0.0.1 "GET / HTTP/1.1" 404 0 "-" "-" 0.000192
[2022-09-09T02:18:53Z INFO  actix_web::middleware::logger] 127.0.0.1 "POST /jobs HTTP/1.1" 200 375 "-" "-" 0.000678
[2022-09-09T02:18:54Z INFO  actix_web::middleware::logger] 127.0.0.1 "GET /jobs/0 HTTP/1.1" 200 431 "-" "-" 0.000406
